cost/flexera/cco/cheaper_regions/cheaper_regions.pt

```diff
@@ -322,18 +366,22 @@ script "js_merge_bc_name", type: "javascript" do
       }
       }
     }
-EOS
+
+    if (results.length > 0) {
+      results[0]['policy_name'] = ds_applied_policy['name']
+    }
+  EOS
 end
 
 ###############################################################################
 # Policy
 ###############################################################################
 
-policy "policy_cheaper_regions" do
-  validate_each $ds_new_bc_cost_name do
-    summary_template "{{ rs_project_name }} (Account ID: {{ rs_project_id }}): {{ len data }} rows with cheaper data"
-    check eq(0,1)
-    escalate $email
+policy "pol_cheaper_regions" do
+  validate $ds_new_bc_cost_name do
+    summary_template "{{ with index data 0 }}{{ .policy_name }}{{ end }}: {{ len data }} Cheaper Regions Found"
```

Comment:
It looks like, after reviewing the incident, that the policy is specifically reporting INSTANCES that can be moved to cheaper regions. The summary_template should always identify specifically the type of resource or item being reported in the incident table. So instead of "Cheaper Regions Found", it should say something like "Instances With Cheaper Region Found".